scripts/run_skeleton.py

```diff
@@ -0,0 +1,52 @@
+from taskqueue import LocalTaskQueue
```

Comment:
Should include some documentation describing what this is if you're not planning on deleting it once we're ready for production.